v8/mastodoncomments/files/assets/js/mastodon.sd.js

```diff
@@ -0,0 +1,144 @@
+const MASTODON_ACCOUNT_ID = '109270094542366763'
+const MASTODON_HOST = 'nerdculture.de'
```

Comment:
Should this be hardcoded here? Where are those `.sd.css` and `.sd.js` files used?

Reply:
They aren't (anymore). They were the results of my first try using the method by sd.ai (mentioned somewhere as inspiration) but it wasn't indented on responses so I tried Carl Schwan's method. They are more similar than distinct as I learned during integration but hindsight is 20/20. I removed them. Maybe in the future I take another look because I like the automatic loading. I recognize that there are valid use-cases to not load the comments immediately so maybe that should be an option.